UPGRADE.md

```diff
@@ -2,6 +2,8 @@
 
 ## dev-develop
 
+When upgrading also have a look at the changes in the [sulu skeleton](https://github.com/sulu/sulu-minimal/compare/2.0.0-alpha5...develop).
```

Comment:
@danrot what you think about this? so we don't spam the  UPGRADE.md but its clearly linked and documented that changes from the skeleton should be added and we add the diff link to every release.

Reply:
That's indeed better :+1: But shouldn't it be `2.0.0-alpha5...2.0.0-alpha4` then?